addons/payment_mollie_official/models/sale_order_line.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from odoo import api, models, fields, _
+from odoo.addons.payment_mollie_official.models.payment_acquirer_method import get_base_url, get_mollie_provider_key
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class SaleOrderLine(models.Model):
```

Comment:
same, can't be there

Reply:
Are you talking about the fields `price_unit_taxinc` and `acquirer_reference`?